src/components/SearchForm.vue

```diff
@@ -98,63 +104,86 @@ export default {
     return {
       searchValue: '',
       adultIncluded: false,
+      region: '',
+      language: '',
+      year: '',
+      page: 1,
     };
   },
   computed: {
-    isNewSearchValue() {
-      return this.searchValue !== this.$route.query.query
+    isNewSearch() {
+      if(JSON.stringify(this.query) === JSON.stringify(this.$route.query)) {
+        return false;
+      } else {
+        return true;
+      }
     },
-    isNewAdultIncluded() {
-      return this.adultIncluded !== this.booleanQueryAdultIncluded
+    regions() {
+      return this.$store.getters['search/getRegions'];
     },
-    isNewSearch() {
-      return this.isNewSearchValue || this.isNewAdultIncluded
+    languages() {
+      return this.$store.getters['search/getLanguages'];
+    },
+    query() {
+      let query = {
+        ...(this.searchValue) && { query: this.searchValue || '' },
```

Comment:
Why you are using the spread operator here ?

Reply:
it will not work without spread operator
I use it to create a condition